### ISSUE TITLE
Markdown Code Blocks Not Colorized

### ISSUE THREAD

**Issue by thombruce** [thombruce/thombruce.github.io#14]
As the title says, markdown code blocks aren't colorized right now.

GitHub styles these appropriately, for instance:

```pug
<template lang="pug">
nav(class='space-x-2')
  ContentNavigation(v-slot="{ navigation }")
    NuxtLink(v-for="link of navigation" :key="link._path" :to="link._path") {{ link.title }}
</template>
```

We're probably just missing a stylesheet from the frontend.

Look into:

- Nuxt code block colorization
- Nuxt content code blocks
- TailwindCSS code block colorization
- Tailwind Typography code blocks

_One of these is likely the culprit, and it can probably be fixed with a simple config tweak._

**Comment by thombruce** [comment]
Trying to turnup some answers to this, I've tried configuring the content highlight settings in `nuxt.config.js` like so:

```json
  content: {
    highlight: {
      theme: 'github-light',
    }
  }
```

I've tried some other variants on that too, but no luck.

I've also looked at the configuration for Nuxt Content's own documentation site, and while it clearly works there as intended there is no special difference in configuration where highlighting is concerned... _Note: They do use Docus, which is documentation theming currently behind beta access request._

The post-processing of code blocks just isn't happening. I've inspected the page source: it's plaintext in there, no rendering by the Shiki library at all.
